tests/tensor/test_extra_ops.py

```diff
@@ -1143,6 +1143,21 @@ def shape_tuple(x, use_bcast=True):
     assert isinstance(b_at[-1].owner.op, Assert)
 
 
+def test_broadcast_shape_constants():
```

Comment:
Should we include conditions where one of the dims is a constant of 1 (and also where both are constants of 1)? Or are those checked in a different test?

Reply:
Yes, those should be covered by some of the other tests, but I added some explicit ones anyway.